_rules/word-spacing-not-important-9e45ec.md

```diff
@@ -0,0 +1,159 @@
+- the [computed][] value is at least `0.16` times its [computed][] [font-size][].
+
+## Assumptions
+
+If there is a mechanism available on the page by which word spacing can be adjusted, failing this rule might not mean [success criterion 1.4.12 Text spacing](https://www.w3.org/TR/WCAG21/#text-spacing) is not satisfied.
+
+## Accessibility Support
+
+While some assistive technologies are able to set [user origin][] styles, others, such as browser extensions, are only able to set style with the [author origin][]. Such assistive technologies can not create styles with a high enough [priority][] to override a `style` attribute with the [important flag][]. If accessibility support does not include assistive technologies that override word spacing through [author origin][], this rule should not be used.
+
+## Background
+
+- [Understanding Success Criterion 1.4.12: Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)
+- [CSS Text Module Level 3 - Spacing](https://www.w3.org/TR/css-text-3/#spacing)
+- [CSS Visual formatting model details](https://drafts.csswg.org/css2/visudet.html)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `strong` element has a `word-spacing` of `1.92pt !important` (equals `0.12em`) which is equal to the recommended minimum, given the specified font size is 16 pixels.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 16px;
+		}
+	</style>
+
+	<body>
+		<strong style="word-spacing: 1.92pt !important;">
+			The toy brought back fond memories of being lost in the rain forest.
+		</strong>
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `strong` element has a `word-spacing` of `2.5px !important` (equals `0.15625em` ) which is below the recommended minimum, given the specified font size is 16 pixels.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 16px;
+		}
+	</style>
+
+	<body>
+		<strong style="word-spacing: 8px !important;">
+			The toy brought back fond memories of being lost in the rain forest.
+		</strong>
+	</body>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+This `p` element is not [visible][] because of `display: none`.
+
+```html
+<p style="display: none">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Inapplicable Example 2
+
+This text is not [visible][] because it is positioned off screen.
+
+```html
+<p style="position: absolute; top: -999em">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Inapplicable Example 3
+
+This `body` element does not have a `style` attribute specified.
+
+```html
+<html>
+	<style>
+		body {
+			word-spacing: 10px;
+		}
+	</style>
+	<body>
+		The toy brought back fond memories of being lost in the rain forest.
+	</body>
+</html>
+```
+
+#### Inapplicable Example 4
+
+This `p` element has a [word-spacing][] of `12px` specified via the style attribute, but does not have the [important flag][].
```

Comment:
Now Passing.

Reply:
Agree again. With similar thoughts on including a passed example with too small a word-spacing and no !important flag, and using fixed units in the examples.